lake/oneshot_cls.py

```diff
@@ -206,7 +205,7 @@ def main():
   study_loader = torch.utils.data.DataLoader(study_dataset, batch_size=config['study_batch_size'], shuffle=False)
   recall_loader = torch.utils.data.DataLoader(recall_dataset, batch_size=config['study_batch_size'], shuffle=False)
 
-  assert len(study_loader) == len(recall_loader)
+  assert len(study_loader)==len(recall_loader)
```

Comment:
Can you restore the spacing here between `)==(`? Not sure if it was removed on accident (seems to be the only change.

Reply:
Noted!